WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -654,11 +654,11 @@ public Height GetBestHeight()
 		}
 	}
 
-	public Height GetBestTurboSyncHeight()
+	public Height GetBestHeight(SyncType syncType)
 	{
 		lock (CriticalStateLock)
 		{
-			return BlockchainState.TurboSyncHeight;
+			return syncType == SyncType.Turbo ? BlockchainState.TurboSyncHeight : BlockchainState.Height;
 		}
 	}
```

Comment:
Can't we merge the two `GetBestHeight` function into one like this?

```
public Height GetBestHeight(SyncType syncType = SyncType.Complete)
{
	lock (CriticalStateLock)
	{
		return syncType == SyncType.Turbo ? BlockchainState.TurboSyncHeight : BlockchainState.Height;
	}
}
```

This shouldn't introduce any new behavior AFAIK.

Reply:
Understandable, it was only an idea.